.github/workflows/R-CMD-check.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
 
       - name: Install libgeos on Linux
         if: runner.os == 'Linux'
-        run: sudo apt-get update -y && sudo apt-get install -y libgeos-dev
+        run: sudo ./tools/ci/install-linux-dependencies.sh
```

Comment:
(nean == Not Even A Nit (TM))  I usually draw at 'too tiny script so I would have kept most shell expressions here, specifically one-liners but it is six of one and half a dozen of another and if you like it better the other way :man_shrugging: